tests/translator/output/implicit_api_with_auth_and_conditions_max.json

```diff
@@ -1,232 +1,251 @@
 {
   "Conditions": {
+    "ServerlessRestApiCondition": {
+      "Fn::Or": [
+        {
+          "Condition": "FunctionCondition"
+        },
+        {
+          "Condition": "FunctionCondition2"
+        },
+        {
+          "Condition": "FunctionCondition3"
+        },
+        {
+          "Condition": "FunctionCondition4"
+        },
+        {
+          "Condition": "FunctionCondition5"
+        },
+        {
+          "Condition": "FunctionCondition6"
+        }
+      ]
+    },
     "FunctionCondition6": {
       "Fn::Equals": [
-        true, 
+        true,
         false
       ]
-    }, 
+    },
     "FunctionCondition5": {
       "Fn::Equals": [
-        true, 
+        true,
         false
       ]
-    }, 
+    },
     "FunctionCondition4": {
       "Fn::Equals": [
-        true, 
+        true,
         false
       ]
-    }, 
+    },
     "FunctionCondition3": {
       "Fn::Equals": [
-        true, 
+        true,
         false
       ]
-    }, 
+    },
     "FunctionCondition2": {
       "Fn::Equals": [
-        true, 
+        true,
         false
       ]
-    }, 
-    "ServerlessRestApiCondition": {
+    },
+    "SamApiPathConditionServerlessRestApiSLASHusers": {
```

Comment:
The `SLASH` is a little distracting. Any ideas for something better?

Reply:
Discussed as a group. Can't think of anything substantially better so will leave it.